.eslintrc

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "airbnb",
   "rules": {
+    "no-var": 0,
     "object-curly-spacing": 0,
-    "no-var": 0
+    "space-before-function-paren": 0
```

Comment:
Yeah - space before '(' looks odd  to me.

Fwiw I've changed things as I've found them but we can always have a discussion as a team about style / lint prefs especially if there's good reason to do things differently. Purely stylistic stuff is harder because it's more subjective but again nothing is ever set in stone.